src/components/dashboard/sidebar/SubscriptionSummaryCard.jsx

```diff
@@ -84,22 +86,87 @@ const SubscriptionSummaryCard = ({
     }
 
     return null;
-  }, [subscriptionPlan, licenseRequest, intl, programProgressPage]);
+  }, [subscriptionPlan, licenseRequest, showExpirationNotifications, intl, programProgressPage]);
+
+  const subscriptionExpirationDate = useMemo(() => {
+    if (!showExpirationNotifications) {
+      return null;
+    }
+    if (!subscriptionPlan) {
+      return (
+        <CardSection>
+          <span>
+            <FormattedMessage
+              id="enterprise.dashboard.sidebar.license.requested.notice"
+              defaultMessage="Awaiting approval."
+              description="License request awaiting approval notice on the enterprise dashboard sidebar."
+            />
+          </span>
+        </CardSection>
+
+      );
+    }
+    const subscriptionDate = dayjs(subscriptionPlan.expirationDate).format('MMMM Do, YYYY');
+    let subscriptionDateLabel;
+    if (subscriptionPlan.isCurrent) {
+      subscriptionDateLabel = intl.formatMessage({
+        id: 'enterprise.dashboard.sidebar.subscription.active.date.prefix.text',
+        defaultMessage: 'Available until',
+        description: 'Subscription active date prefix on the enterprise dashboard sidebar.',
+      });
+    } else {
+      subscriptionDateLabel = intl.formatMessage({
+        id: 'enterprise.dashboard.sidebar.subscription.expired.date.prefix',
+        defaultMessage: 'Expired on',
+        description: 'Subscription expired date prefix on the enterprise dashboard sidebar.',
+      });
+    }
+    return (
+      <CardSection>
+        {subscriptionDateLabel}
+        {' '}
+        <span
+          className="font-weight-bold"
+        >
+          {subscriptionDate}
+        </span>
+      </CardSection>
+    );
+  }, [intl, showExpirationNotifications, subscriptionPlan]);
+
+  const programProgressSubscriptionExpirationWarningModal = useMemo(() => {
+    if (!showExpirationNotifications && courseEndDate > subscriptionPlan.expirationDate) {
```

Comment:
I'm a bit confused by this condition because I feel like before it DID render a modal if courseEndDate > subscriptionPlan.expirationDate, but now it doesn't?

Reply:
Yep, good catch here too with the conditional. 👍🏽